goal_src/jak2/engine/collide/collide-cache.gc

```diff
@@ -219,7 +219,7 @@
   )
 
 (defmethod fill-using-line-sphere collide-cache ((obj collide-cache) (arg0 collide-query))
-  (local-vars (v1-11 uint) (v1-20 float))
+  (local-vars (v1-11 float) (v1-20 float))
```

Comment:
I think this one should stay a `uint` (sorry, I should have left a comment here)